apps/andi/lib/andi_web/views/error_helpers.ex

```diff
@@ -14,15 +14,22 @@ defmodule AndiWeb.ErrorHelpers do
   def error_tag(form, _, _) when not is_map(form), do: []
 
   def error_tag(form, field, options) do
-    Enum.map(Keyword.get_values(form.errors, field), fn error ->
-      translated = error |> interpret_error(field) |> translate_error()
-
-      content_tag(:span, translated,
-        class: "error-msg",
-        id: "#{field}-error-msg",
-        data: get_additional_content_tag_data(form, field, options)
-      )
-    end)
+    form.errors
+    |> Map.new()
+    |> Map.get(field)
+    |> generate_error_tag(field, form, options)
```

Comment:
I put this here to prevent us from showing multiple error messages for the same field. I don't know of a case where we would want more than one error message in the same form but if there is one I can change this back

Reply:
That's probably fine